### PR TITLE
Temporal Nexus: References (Commands, Errors, Failures) 

### PR DESCRIPTION
**Pull request by jsundai**
## Notes to reviewers
additional PR created for events

[Preview ](https://temporal-documentation-git-nexus-references.preview.thundergun.io/references/)

### CODE REVIEW

File: docs/references/errors.mdx [hunk 1]
```diff
@@ -100,6 +100,12 @@ This error indicates unset or invalid attributes for [`ScheduleActivityTask`](/r
 Reset any unset or empty attributes.
 Adjust the size of the received payload to stay within the given size limit.
 
+## Bad Schedule Nexus Operation Attributes
+
+This error indicates unset or invalid attributes for ScheduleNexusOperation, for example if the Nexus Endpoint name used in the caller Workflow doesn't exist.
+
+Reset any unset or empty attributes and adjust the size of the received payload to stay within the given size limit.
```
Comment (bergundy) [inline docs/references/errors.mdx:107]:
```suggestion
Inspect the reason given in the error for mitigation when possible.
```

File: docs/references/failures.mdx [hunk 2]
```diff
@@ -82,26 +82,130 @@ During conversion, the following Application Failure fields are set:
 
 When an [Activity Execution](/activities#activity-execution) fails, the Application Failure from the last Activity Task is the `cause` field of the [ActivityFailure](#activity-failure) thrown in the Workflow.
 
+### Errors in Nexus Operations
+
+An error in a Nexus Operation can cause either a Nexus Operation Task Failure (the Task will be retried) or a Nexus Operation Execution Failure (the Nexus Operation is marked as failed).
```
Comment (bergundy) [inline docs/references/failures.mdx:87]:
```suggestion
Nexus Operations can end up in completed, failed, canceled, and timed out states.

Under the hood, the Nexus Operation machinery, breaks up the lifecycle of an Operation into one or more StartOperation requests and completion callbacks and automatically retries these requests as long they fail with retryable errors.

The Workflow-specified schedule-to-close timeout is enforced by the caller's machinery and the only way for an Operation to transition to the timed out state.

Operations can end up in the other three states either when a user handler returns a synchronous response or error, or when an asynchronous Operation (like one backed by a workflow) eventually reaches a terminal state.

A handler can return either retryable or non-retryable errors to indicate to the caller's Nexus machinery whether to retry a given request. Requests that time out before a response is sent to the caller are automatically retried.

By default, errors are considered retryable, unless specified below:

- Non retryable Application Failures
- Unsuccessful Operation errors that can resolve an operation as either failed or canceled
- [Handler errors](https://github.com/nexus-rpc/api/blob/main/SPEC.md#predefined-handler-errors) with the following types: `BAD_REQUEST`, `UNAUTHENTICATED`, `UNAUTHORIZED`, `NOT_FOUND`, and `RESOURCE_EXHAUSTED`


```

Comment (bergundy) [reply docs/references/failures.mdx:87]: Can replace all of this content with this suggestion, it's more concise, accurate and doesn't go too much into implementation details.

File: docs/references/failures.mdx [hunk 3]
```diff
@@ -122,6 +226,29 @@ For example, if an Activity Execution times out, the `cause` is a [Timeout Failu
 - Python: [ActivityError](https://python.temporal.io/temporalio.exceptions.ActivityError.html)
 - Proto: [ActivityFailureInfo](https://api-docs.temporal.io/#temporal.api.failure.v1.ActivityFailureInfo) and [Failure](https://api-docs.temporal.io/#temporal.api.failure.v1.Failure)
 
+## Nexus Operation Failure
```
Comment (bergundy) [inline docs/references/failures.mdx:229]: This is great!

**Review by fairlydurable — APPROVED** [review]
Unblocking this so it can move forward. Any style issues can be addressed in follow-up tickets.

**Pull request merged**